examples/transloadit/main.js

```diff
@@ -45,81 +68,149 @@ formUppy.on('upload-error', (file, err) => {
     .textContent = err.message
 })
 
+formUppy.on('complete', ({ transloadit }) => {
+  const btn = document.getElementById('uppy-select-files')
+  btn.hidden = true
+  const selectedFiles = document.getElementById('uppy-form-selected-files')
+  selectedFiles.textContent = `selected files: ${Object.keys(transloadit[0].results).length}`
+})
+
 window.formUppy = formUppy
 
-const formUppyWithDashboard = robodog.form('#dashboard-form', {
+/**
+ * Form with Dashboard
+ */
+
+const formUppyWithDashboard = new Uppy({
   debug: true,
-  fields: ['message'],
+  autoProceed: false,
   restrictions: {
     allowedFileTypes: ['.png'],
   },
-  waitForEncoding: true,
-  note: 'Only PNG files please!',
-  params: {
-    auth: { key: TRANSLOADIT_KEY },
-    template_id: TEMPLATE_ID,
-  },
-  dashboard: '#dashboard-form .dashboard',
 })
+  .use(Dashboard, {
+    inline: true,
+    target: '#dashboard-form .dashboard',
+    note: 'Only PNG files please!',
+    hideUploadButton: true,
+  })
+  .use(RemoteSources, { companionUrl: COMPANION_URL })
+  .use(Form, {
+    target: '#dashboard-form',
+    fields: ['message'],
+    triggerUploadOnSubmit: true,
+    submitOnSuccess: true,
+    addResultToForm: true,
+  })
+  .use(Transloadit, {
+    waitForEncoding: true,
+    params: {
+      auth: { key: TRANSLOADIT_KEY },
+      template_id: TEMPLATE_ID,
+    },
+  })
 
 window.formUppyWithDashboard = formUppyWithDashboard
 
-const dashboard = robodog.dashboard('#dashboard', {
+/**
+ * Dashboard
```

Comment:
If every example shows Dashboard, then it feels a bit weird to now do that again?